ci/linkcheck.sh

```diff
@@ -42,7 +46,7 @@ fi
 echo "exec $LINKCHECK_BINARY $FLAGS"
 if [ "$USE_TOKEN" = 1 ]; then
   config=$(set_github_token)
-  exec $LINKCHECK_BINARY $FLAGS <<<"$config"
+  exec env MDBOOK_OUTPUT__LINKCHECK__FOLLOW_WEB_LINKS=$FOLLOW_WEB_LINKS $LINKCHECK_BINARY $FLAGS <<<"$config"
 else
-  exec $LINKCHECK_BINARY $FLAGS
+  exec env MDBOOK_OUTPUT__LINKCHECK__FOLLOW_WEB_LINKS=$FOLLOW_WEB_LINKS $LINKCHECK_BINARY $FLAGS
```

Comment:
I honestly don't know if this shell incantation is correct... Does this look right to you?

Reply:
It has to be true/false, not 0/1, otherwise it fails :laughing: Otherwise looks fine.

Reply:
... Oh.

Reply:
Hopefully I fixed it :D